src/hooks/screen/useChatScreenHook.ts

```diff
@@ -52,13 +74,15 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       return;
     }
 
+    const replyData = replyPreview;
+    let json;
     let currentChatSchema = sendingChatSchema;
     let userId = await getUserId();
     const myAnonymousId = await getAnonymousUserId();
 
-    if (type === 'ANONYMOUS') {
-      userId = myAnonymousId;
-    }
+    if (type === ANONYMOUS) userId = myAnonymousId;
+    if (replyData) json = {reply_data: replyData};
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The handling of reply data is updated. Make sure that the `reply_data` property is properly handled in all relevant parts of the application, including database schemas and any interfaces or types that might be affected.
```diff
+ const replyData = replyPreview;
+ let json;
+ let currentChatSchema = sendingChatSchema;
+ let userId = await getUserId();
+ const myAnonymousId = await getAnonymousUserId();
+ 
+ if (type === ANONYMOUS) userId = myAnonymousId;
+ if (replyData) json = {reply_data: replyData};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->